assets/src/edit-story/elements/utils/getCommonAttributes.js

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Remove per the latest changes in #137

Reply:
That was quite the merge conflict, but I managed it!